pkg/controller/certificates/csr.go

```diff
@@ -109,7 +111,11 @@ func GetCertificatesAPIVersion(clientSet kubernetes.Interface) CSRVersion {
 func GetCSRSignerName(clientSet kubernetes.Interface) string {
 	csrSignerNameOnce.Do(func() {
 		// At the moment we will use kubernetes.io/kubelet-serving as the default
-		csrSignerName = getDefaultCsrSignerName()
+		var csrSignerNameFromEnv bool
+		csrSignerName, csrSignerNameFromEnv = getDefaultCsrSignerName()
+		if csrSignerNameFromEnv {
```

Comment:
I think we can just say like 
```
if os.Getenv(CSRSignerName) != "" {
    return
}
```
and no need to change the function signature for `getDefaultCsrSignerName`